src/components/SearchableMultiSelect/SearchableMultiSelect.jsx

```diff
@@ -549,12 +563,13 @@ const SearchableMultiSelect = createClass({
 				className={cx('&', className)}
 			>
 				<DropMenu
+					Error={errorChildProps}
```

Comment:
Does it make sense to pass an `Error` prop here? It doesn't seem like DropMenu takes an `Error` prop.

Reply:
I wasn't entirely sure where to pass in the error props; I initially wrapped the entire component in a div and passed the error props in that container but Jon indicated that it would cause some React errors in the console. I was unsure where to pass the `Error` prop outside of the div since the other components do not take an Error prop.

Reply:
@dreidn I am realizing that the `Error` prop is not necessary in any of the components because they are not expecting it. I am removing passing the `Error` prop and simply relying on the errorChildProps driving the validation logic in all three of the components. Forgive my mistakes :)